src/elements/content-uploader/ItemRemove.js

```diff
@@ -33,13 +34,20 @@ const ItemRemove = ({ onClick, status }: Props) => {
 
     return (
         <div className="bcu-item-action">
-            <Tooltip position="top-left" text={<FormattedMessage {...messages.remove} />}>
-                <PlainButton onClick={onClick} type="button" isDisabled={status === STATUS_STAGED} {...resin}>
+            <Tooltip position="top-left" text={intl.formatMessage(messages.remove)}>
+                <PlainButton
+                    aria-label={intl.formatMessage(messages.remove)}
+                    isDisabled={status === STATUS_STAGED}
```

Comment:
you could update this to include the similar feedback from the ItemAction PR

Reply:
yes i can!